web/wp-content/themes/dctx-v2/template-parts/scaffolding/scaffolding-globals.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-etrk9d.php, but found phpcs-scan-ETRk9d.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).